### PR TITLE
Update msg_hash_fr.h

### PR DESCRIPTION
**Pull request by BlaCkMatiK**
Hi ! French guy here, I hope what i made is right and worked, please tell me if i'm wrong ! :)
I don't know how it works but, i hope i'm in the right place to help ? 
If no, write me to lalannemateo@hotmail.com to tell me how can i improve the translation :)

Cordially ?? BlaCkMatiK

### CODE REVIEW

File: intl/msg_hash_fr.h [hunk 1]
```diff
@@ -3155,46 +3157,46 @@ MSG_HASH(MSG_INPUT_ENABLE_SETTINGS_PASSWORD_NOK,
...
 MSG_HASH(MENU_ENUM_SUBLABEL_PLAYLIST_ENTRY_RENAME,
       "Autoriser l'utilisateur à renommer les entrées des playlists.")
 MSG_HASH(MENU_ENUM_LABEL_VALUE_PLAYLIST_ENTRY_RENAME,
       "Autoriser le renommage des entrées")
 MSG_HASH(MENU_ENUM_LABEL_VALUE_MENU_SHOW_LOAD_CORE,
-      "Show Load Core")
+      "Afficher Charger un cœur")
 MSG_HASH(MENU_ENUM_SUBLABEL_MENU_SHOW_LOAD_CORE,
-      "Show/hide the 'Load Core' option.")
+      "Afficher/Cacher l'option 'Charger un cœur' .")
```
Comment (orbea) [inline intl/msg_hash_fr.h]: Extra trailing space?

File: intl/msg_hash_fr.h [hunk 2]
```diff
@@ -2969,119 +2969,121 @@ MSG_HASH(MENU_ENUM_SUBLABEL_DPI_OVERRIDE_ENABLE,
...
-      "Custom viewport width that is used if the Aspect Ratio is set to 'Custom'.")
+      "Largeur de la fenêtre d'affichage utilisée si le rapport d'aspect est défini sur 'Personnalisé'.")
 MSG_HASH(MENU_ENUM_SUBLABEL_VIDEO_VIEWPORT_CUSTOM_X,
-      "Custom viewport offset used for defining the X-axis position of the viewport. These are ignored if 'Integer Scale' is enabled. It will be automatically centered then.")
+      "Décalage de la fenêtre d'affichage personnalisé utilisé pour définir la position de l'axe X de la fenêtre. Ceux-ci sont ignorés si 'Zoom Entier' est activé. Il sera ensuite automatiquement centré
+.")
 MSG_HASH(MENU_ENUM_SUBLABEL_VIDEO_VIEWPORT_CUSTOM_Y,
-      "Custom viewport offset used for defining the Y-axis position of the viewport. These are ignored if 'Integer Scale' is enabled. It will be automatically centered then.")
+      "Décalage de la fenêtre d'affichage personnalisé utilisé pour définir la position de l'axe Y de la fenêtre. Ceux-ci sont ignorés si 'Zoom Entier' est activé. Il sera ensuite automatiquement centré
+.")
```
Comment (orbea) [inline intl/msg_hash_fr.h]: Extra new line here.

File: intl/msg_hash_fr.h [hunk 3]
```diff
@@ -2969,119 +2969,121 @@ MSG_HASH(MENU_ENUM_SUBLABEL_DPI_OVERRIDE_ENABLE,
...
 MSG_HASH(MENU_ENUM_SUBLABEL_VIDEO_VIEWPORT_CUSTOM_HEIGHT,
-      "Custom viewport height that is used if the Aspect Ratio is set to 'Custom'.")
+      "Hauteur de la fenêtre d'affichage utilisée si le rapport d'aspect est défini sur 'Personnalisé'.")
 MSG_HASH(MENU_ENUM_SUBLABEL_VIDEO_VIEWPORT_CUSTOM_WIDTH,
-      "Custom viewport width that is used if the Aspect Ratio is set to 'Custom'.")
+      "Largeur de la fenêtre d'affichage utilisée si le rapport d'aspect est défini sur 'Personnalisé'.")
 MSG_HASH(MENU_ENUM_SUBLABEL_VIDEO_VIEWPORT_CUSTOM_X,
-      "Custom viewport offset used for defining the X-axis position of the viewport. These are ignored if 'Integer Scale' is enabled. It will be automatically centered then.")
+      "Décalage de la fenêtre d'affichage personnalisé utilisé pour définir la position de l'axe X de la fenêtre. Ceux-ci sont ignorés si 'Zoom Entier' est activé. Il sera ensuite automatiquement centré
+.")
```
Comment (orbea) [inline intl/msg_hash_fr.h]: Extra new line here.

File: intl/msg_hash_fr.h [hunk 4]
```diff
@@ -2969,119 +2969,121 @@ MSG_HASH(MENU_ENUM_SUBLABEL_DPI_OVERRIDE_ENABLE,
...
 MSG_HASH(MENU_ENUM_SUBLABEL_VIDEO_FONT_PATH,
       "Choisir une police différente pour les notifications.")
 MSG_HASH(MENU_ENUM_SUBLABEL_SHADER_APPLY_CHANGES,
-      "Changes to the shader configuration will take effect immediately. Use this if you changed the amount of shader passes, filtering, FBO scale, etc.")
+      "Les changements de la configuration des Shaders prendront effet immédiatement. Utilisez cela si vous avez changé le nombre de passages, la filtration, son zoom , etc.")
 MSG_HASH(
       MENU_ENUM_SUBLABEL_VIDEO_SHADER_NUM_PASSES,
-      "Increase or decrease the amount of shader pipeline passes. You can bind a separate shader to each pipeline pass and configure its scale and filtering."
+      "Augmenter ou diminuer le nombre de passages du Shader. Vous pouvez lier un Shader différent à chaque passage et configurer son zomm et sa filtration.")
       )
```
Comment (orbea) [inline intl/msg_hash_fr.h]: Extra parenthesis.

File: intl/msg_hash_fr.h [hunk 5]
```diff
@@ -2969,119 +2969,121 @@ MSG_HASH(MENU_ENUM_SUBLABEL_DPI_OVERRIDE_ENABLE,
...
-      "Dossier d'emplacement des captures d'écran.")
+      "Répertoire où sont stockées les captures d'écran.")
 MSG_HASH(MENU_ENUM_SUBLABEL_OVERLAY_DIRECTORY,
-      "Defines a directory where overlays are kept for easy access.")
+      "Répertoire où sont stockés les overlays, pour un accès rapide.")
 MSG_HASH(
       MENU_ENUM_SUBLABEL_CHEAT_DATABASE_PATH,
-      "Les fichiers de triche sont stockés ici."
+      "Répertoire où sont stockés les fichiers de triche.")
       )
```
Comment (orbea) [inline intl/msg_hash_fr.h]: Extra parenthesis.

File: intl/msg_hash_fr.h [hunk 6]
```diff
@@ -2969,119 +2969,121 @@ MSG_HASH(MENU_ENUM_SUBLABEL_DPI_OVERRIDE_ENABLE,
...
-      "Saved queries are stored to this directory.")
+      "Les curseurs enregistrés sont stockés dans ce répertoire.")
 MSG_HASH(
       MENU_ENUM_SUBLABEL_CONTENT_DATABASE_DIRECTORY,
       "Les bases de données sont stockées dans ce répertoire."
       )
 MSG_HASH(
       MENU_ENUM_SUBLABEL_ASSETS_DIRECTORY,
-      "This location is queried by default when menu interfaces try to look for loadable assets, etc."
+      "Ce répertoire est inspecté par défaut lorque les interfaces du menu cherchent des assets, etc. ."
```
Comment (orbea) [inline intl/msg_hash_fr.h]: Extra trailing space.

File: intl/msg_hash_fr.h [hunk 7]
```diff
@@ -2908,7 +2908,7 @@ MSG_HASH(
 MSG_HASH(MENU_ENUM_SUBLABEL_DYNAMIC_WALLPAPERS_DIRECTORY,
       "Directory to store wallpapers dynamically loaded by the menu depending on context.")
 MSG_HASH(MENU_ENUM_SUBLABEL_THUMBNAILS_DIRECTORY,
-      "Supplementary thumbnails (boxarts/misc. images, etc.) are stored here."
+      "Vignettes supplémentaires (jaquettes/images diverses, etc.) sont stockées ici"
```
Comment (orbea) [inline intl/msg_hash_fr.h]: Missing period? Seems inconsistent compared to other strings.

**Comment by orbea** [comment]
Seems the build is not passing so I left various comments on possible issues.

**Comment by twinaphex** [comment]
Some checks failed as you can see above -

[2019-04-25 11:38:19] [build] intl/msg_hash_fr.h:3563: error: unterminated argument list invoking macro "MSG_HASH"
[2019-04-25 11:38:19] [build]      )
[2019-04-25 11:38:19] [build]  
[2019-04-25 11:38:19] [build] intl/msg_hash_fr.h:2817:1: error: ‘MSG_HASH’ undeclared (first use in this function); did you mean ‘MSG_LAST’?

**Comment by BlaCkMatiK** [comment]
> Some checks failed as you can see above -
> 
> [2019-04-25 11:38:19] [build] intl/msg_hash_fr.h:3563: error: unterminated argument list invoking macro "MSG_HASH"
> [2019-04-25 11:38:19] [build] )
> [2019-04-25 11:38:19] [build]
> [2019-04-25 11:38:19] [build] intl/msg_hash_fr.h:2817:1: error: ‘MSG_HASH’ undeclared (first use in this function); did you mean ‘MSG_LAST’?

Yeah i saw that but i don't understand at all wat's wrong ? I think i'm a little idiot on this part :/ ? Can you help me ?

**Comment by WeedyWeedSmoker** [comment]
@BlaCkMatiK Don't worry, I'm re-doin' it right now...

@twinaphex I've taken the translation over, I'm correcting bugs, inconsistencies, and started from the newer more complete US files ! Translating the missing parts too...

**Comment by WeedyWeedSmoker** [comment]
@BlaCkMatiK The complete translation is now done... You can check it here :

https://github.com/libretro/RetroArch/pull/8670

And you can close this PR if it's ok for you !

**Comment by twinaphex** [comment]
OK @WeedyWeedSmoker so this can be closed now because the other PR has been merged instead?

**Comment by WeedyWeedSmoker** [comment]
@twinaphex Yeah, I confirmed with @BlaCkMatiK, I think he just forgot to close it...

**Pull request closed without merging**